code/procs/mobprocs/chatprocs.dm

```diff
@@ -717,7 +713,7 @@
 		looc_style = "color: #a24cff;"
 
 	var/image/chat_maptext/looc_text = null
-	looc_text = make_chat_maptext(src, "\[LOOC: [msg]]", looc_style)
+	looc_text = make_chat_maptext(our_location, "\[LOOC: [msg]]", looc_style)
```

Comment:
would this make the floating chat not 'follow' the mob if they move?
If so, maybe only use `our_location` if `src.loc != our_location`

Reply:
good observation,

this was an initial fix, with the assumption that looc messages were hiding because they were being placed inside the container

but the actual issue was searching for mobs and assigning them as recipients to assign visibility to the maptext.

just putting it back to src and completely removing our_location works fine